docker/readme.md

```diff
@@ -0,0 +1,15 @@
+# How to run this
+
+Change url and token in Dockerfile
+
+```bash
+docker build . -t gitlab-cli
+docker run -v $PWD:/home -ti --rm gitlab-cli merge-request -b docs/swagger -t v1.3.0 -v -a USERNAME -m MR_Title
```

Comment:
You may replace `docs/swagger` with BRANCH_NAME to make it generic.

Reply:
ok, did the same with the tag :+1: